spras/netmix2.py

```diff
@@ -0,0 +1,103 @@
+from pathlib import Path
+
+from spras.containers import prepare_volume, run_container
+from spras.dataset import Dataset
+from spras.prm import PRM
+
+__all__ = ['NetMix2']
+
+class NetMix2(PRM):
+    required_inputs = ['network', 'scores']
+
+    @staticmethod
+    def generate_inputs(data: Dataset, filename_map):
```

Comment:
For directionality and edges, I just wanted to confirm that NetMix2 deals with only undirected inputs and can handle duplicate edges?

Reply:
[Undirected](https://github.com/raphael-group/netmix2/blob/110954e27ec1911b6fe68b6a34352fecca494922/src/common.py#L39-L40).